### PR TITLE
Limited docker version - lower than 3.0.0

### LINKED ISSUE #869: Python Docker client 3.0 breaking change
##### ISSUE TYPE

 - Bug Report

##### SUMMARY

Version [3.0.0 of the Python `docker` client was released last night](https://pypi.python.org/pypi/docker/3.0.0) which seems to have broken the conductor build process.

All of our `ansible-container` projects are failing to build this morning.

Pinning `docker` below 3.0 resolved the issue.

##### STEPS TO REPRODUCE

Force installation of new `docker` client:
`pip install -U ansible-container`

Build any container project with `--no-cache`

##### EXPECTED RESULTS

Containers build as usual

##### ACTUAL RESULTS

Conductor fails to build

```
Building Docker Engine context...	
Starting Docker build of Ansible Container Conductor image (please be patient)...	
Post stage
ERROR	Unknown exception	
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/container/cli.py", line 299, in __call__
    getattr(core, u'hostcmd_{}'.format(args.subcommand))(**vars(args))
  File "/usr/local/lib/python2.7/site-packages/container/__init__.py", line 28, in __wrapped__
    return fn(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/container/core.py", line 181, in hostcmd_build
    environment=env_vars
  File "/usr/local/lib/python2.7/site-packages/container/docker/engine.py", line 105, in __wrapped__
    return fn(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/container/__init__.py", line 28, in __wrapped__
    return fn(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/container/docker/engine.py", line 1070, in build_conductor_image
    return image.id
AttributeError: 'tuple' object has no attribute 'id'
```

### PR DESCRIPTION
**Pull request by amitzig**
Fixes #869 

##### ISSUE TYPE

 - Bugfix Pull Request

##### SUMMARY

```
The change limits the version of docker installed by pip to be lower than 3.0.0
Becuase release 3.0.0 breaks ansible-container
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -84,11 +84,11 @@ def run(self):
         tests_require=[
             'ansible>=2.3.0',
             'pytest>=3',
-            'docker>=2.4.0',
+            'docker>=2.4.0,<=2.7.0',
```
Comment (TomasTomecek) [inline setup.py]: wouldn't `docker<3.0.0` be better? in case there is a bugfix release for `2.7`

**Comment by amitzig** [comment]
Agree with comment, made required changes

**Review by TomasTomecek — APPROVED** [review]
LGTM

**Comment by marcusianlevine** [comment]
This is a duplicate of #870

**Comment by ehelms** [comment]
I merged the original PR (https://github.com/ansible/ansible-container/pull/870) to unblock folks.

**Pull request closed without merging**